spec/lib/tasks/import_spec.rb

```diff
@@ -1,8 +1,9 @@
 RSpec.describe "Import tasks" do
   include ActiveJob::TestHelper
+  include Rails.application.routes.url_helpers
```

Comment:
I don't think think this will be doing anything

Reply:
Removed, which had no negative effect.